integration-test.Rmd

```diff
@@ -190,43 +190,29 @@ Ensure this other configuration file also exists:
 
 ```{r}
 config_2 <- here("parameter_files", "WebParameters_2dii.yml")
-config_2_copy <- tempfile()
-fs::file_copy(config_2, config_2_copy)
```

Comment:
We no longer need to copy the configuration file, change it, and restore it at the end.